src/components/timecard/edit-shift-hours/EditShiftHours.jsx

```diff
@@ -140,16 +160,62 @@ const EditShiftHours = ({
           setShowEditShiftHours(false);
         }
       },
+      true,
       handleServerValidationErrors
     );
   };
 
+  const getCombinedErrors = () => {
+    const existingErrors =
+      Object.keys(errors).length > 0 ? errors : summaryErrors;
+
+    const combinedErrors = { ...existingErrors };
+
+    if (clashingProperty === 'startAndEndTime') {
```

Comment:
Can we extract this string and the other two below into somewhere common and then use those constants here and also in the `handleServerValidationErrors` function.

Reply:
👍 done